static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -282,6 +285,25 @@ const makeABTestVariant = (
             iframeId,
         },
 
+        canRun() {
+            const {
+                count: maxViewCount,
+                days: maxViewDays,
+                minDaysBetweenViews: minViewDays,
+            } = maxViews;
+
+            const testId = parentTest.useLocalViewLog
```

Comment:
what does `useLocalViewLog` mean?

Reply:
good question! this is actually existing functionality, I've just moved it. it just means that it will check views of that specific test, rather than views of all epic tests. so you may have maxed out your global epic view count but you still have views left of this specific test.